File: benchmarks/fixtures/polyglot-services/src/user_service.ml
module UserService = struct
    type user = { name: string; email: string }
    
    let create name email = { name; email }
    
    let display u = Printf.sprintf "%s <%s>" u.name u.email
end

let user = UserService.create "Eve" "eve@example.com" in
print_endline (UserService.display user)

File: benchmarks/fixtures/polyglot-services/src/UserService.fs
module UserService

type User = { Name: string; Email: string }

let display (user: User) = 
    $"{user.Name} <{user.Email}>"

let create name email = 
    { Name = name; Email = email }

let user = create "Diana" "diana@example.com"
printfn "%s" (display user)